utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/kafka/KafkaConnector.java

```diff
@@ -85,17 +86,16 @@ public void produceMessageWithoutNewThread(String message) throws Exception {
     }
 
     // For APM testing, a consume message without starting a new thread
-    public boolean consumeMessageWithoutNewThread(Integer timeout_s) throws Exception {
-        KafkaConsumer<String, String> consumer = createKafkaConsumer();
+    public boolean consumeMessageWithoutNewThread(Integer timeout_s, String topic) throws Exception {
+        KafkaConsumer<String, String> consumer = createKafkaConsumer(topic);
```

Comment:
we already pass the topic to the ctor, I think we should reuse `this.topic` rather than asking for it again as a parameter
(same remark for `startConsumingMessages` above)

Reply:
Good idea, will revise